common/src/main/scala/db/SqlRegistrationRepository.scala

```diff
@@ -87,18 +87,19 @@ class SqlRegistrationRepository[F[_]: Async](xa: Transactor[F])
       )
       ++ fr"GROUP BY token, platform"
       )
-      .query[(String, String)]
+      .query[(String, String, String)]
```

Comment:
Here it would have to be an Option[String] as there's no guarantee the buildTier will be present in the database. In fact we know already some records won't have it in the database